fluentlenium-core/src/main/java/org/fluentlenium/core/Fluent.java

```diff
@@ -360,6 +360,32 @@ public Fluent goTo(String url) {
         return this;
     }
 
+    /**
+     * Open the url page in a new tab
+     *
+     * @param url
+     */
+    public Fluent goToInNewTab(String url) {
+        if (url == null) {
+            throw new IllegalArgumentException("Url is mandatory");
+        }
+
+        if (baseUrl != null) {
+            URI uri = URI.create(url);
+            if (!uri.isAbsolute()) {
+                url = baseUrl + url;
```

Comment:
I'm not sure if you want to always use base URL as a part of URL construction.
Java has gently mechanism to compose URL instead of String adding.

URL myURL = new URL("http://example.com/pages/");
URL page1URL = new URL(myURL, "page1.html");
URL page2URL = new URL(myURL, "page2.html");

Could you re-factor this method to do it in the better way?